src/BPEssentials/Commands/KitCreate.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.Entities;
+using System.IO;
+
+namespace BPEssentials.Commands
+{
+    public class KitCreate : Command
+    {
+        public void Invoke(ShPlayer player, string kit, int price = 0, int delay = 0)
+        {
+            if (delay < 0)
+            {
+                player.TS("delay_error_negative");
+                return;
+            }
+            if (price < 0)
+            {
+                player.TS("price_error_negative");
+                return;
+            }
+            var file = Path.Combine(Core.Instance.Paths.KitsFolder, $"{kit}.json");
+            if (File.Exists(file))
+            {
+                player.TS("expFileHandler_create_error_alreadyExists", player.T(Core.Instance.KitHandler.Name), kit);
+                return;
+            }
+            var obj = new KitHandler.JsonModel
+            {
+                Delay = delay < 0 ? 0 : price,
```

Comment:
This should be `delay`, not `price`.
```suggestion
                Delay = delay < 0 ? 0 : delay,
```